### PR TITLE
Payment status/yagna/stub

### PR DESCRIPTION
**Pull request by tworec**
`yagna payment status` stub tu unlock `golemsp status` impl

```
$ cargo run payment status --help
yagna-payment-status 0.1.0

USAGE:
    yagna payment status [FLAGS] [OPTIONS]

FLAGS:
        --json       Return results in JSON format
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
        --account <account>     
        --driver <driver>        [default: zksync]
        --network <network>     
        --token <token>         
    -d, --datadir <data-dir>    Service data dir [env: YAGNA_DATADIR]  [default: /Users/tworec/Library/Application
                                Support/GolemFactory.yagna]
    -g, --gsb-url <gsb-url>     Service Bus (aka GSB) URL [env: GSB_URL]  [default: tcp://127.0.0.1:7464]

```

### CODE REVIEW

File: core/payment-driver/gnt/src/lib.rs [hunk 1]
```diff
@@ -54,9 +54,9 @@ const CREATE_FAUCET_FUNCTION: &str = "create";
 
 pub const DRIVER_NAME: &'static str = "erc20";
 
-pub const DEFAULT_NETWORK: &'static str = "rinkeby";
-pub const DEFAULT_TOKEN: &'static str = "tGLM";
-pub const DEFAULT_PLATFORM: &'static str = "erc20-rinkeby-tglm";
+pub const DEFAULT_NETWORK: &'static str = "mainnet";
```
Comment (Wiezzel) [inline core/payment-driver/gnt/src/lib.rs:57]: Mainnet is **not** going to be the default. Please revert this file.

File: core/payment-driver/zksync/src/lib.rs [hunk 2]
```diff
@@ -8,9 +8,9 @@
 pub const DRIVER_NAME: &'static str = "zksync";
 pub const ZKSYNC_TOKEN_NAME: &'static str = "GNT";
 
-pub const DEFAULT_NETWORK: &'static str = "rinkeby";
-pub const DEFAULT_TOKEN: &'static str = "tGLM";
-pub const DEFAULT_PLATFORM: &'static str = "zksync-rinkeby-tglm";
+pub const DEFAULT_NETWORK: &'static str = "mainnet";
```
Comment (Wiezzel) [inline core/payment-driver/zksync/src/lib.rs:11]: Mainnet is **not** going to be the default. Please revert this file.

File: core/payment/src/cli.rs [hunk 3]
```diff
@@ -18,12 +18,17 @@ pub enum PaymentCli {
         #[structopt(long, default_value = DEFAULT_PAYMENT_DRIVER)]
         driver: String,
         #[structopt(long)]
-        network: Option<String>,
+        network: Option<String>, // TODO: network enum
```
Comment (Wiezzel) [inline core/payment/src/cli.rs:21]: Payment driver implementors will have absolute freedom defining networks and tokens. There's **not** going to be a network enum nor token enum.

File: core/payment/src/cli.rs [hunk 4]
```diff
@@ -18,12 +18,17 @@ pub enum PaymentCli {
...
     Status {
-        address: Option<String>,
-        #[structopt(long, short)]
-        platform: Option<String>,
+        #[structopt(long)]
+        account: Option<String>,
+        #[structopt(long, default_value = DEFAULT_PAYMENT_DRIVER)]
+        driver: String,
+        #[structopt(long)]
+        network: Option<String>, // TODO: network enum
```
Comment (Wiezzel) [inline core/payment/src/cli.rs:29]: Ditto.

File: core/payment/src/cli.rs [hunk 5]
```diff
@@ -18,12 +18,17 @@ pub enum PaymentCli {
...
+        #[structopt(long)]
+        token: Option<String>, // TODO: token enum?
```
Comment (Wiezzel) [inline core/payment/src/cli.rs:31]: Ditto.

**Review by Wiezzel — CHANGES_REQUESTED** [review]

File: core/payment/src/cli.rs (hunk 3 above)
Comment (tworec) [reply core/payment/src/cli.rs:21]: I'd argue but not at this stage. :)

**Pull request merged**